docs/source/routing/security/cors.mdx

```diff
@@ -67,8 +68,9 @@ To allow browsers to pass credentials to the router, set `allow_credentials` to
 ```yaml {6} title="router.yaml"
 cors:
   origins:
-    - https://www.your-app.example.com
-    - https://studio.apollographql.com
+    - origins:
```

Comment:
Hardest part of comp sci is naming things.  I think customers may benefit if a different name was come up with for one of the two "origin" names at the different levels.  It's caused me confusion in reading the code and config.